### PR TITLE
feat(action): fix empty not-updated section

### PR DESCRIPTION
**Pull request by IamAbbey**

### CODE REVIEW

File: actions_helper/main.py [hunk 1]
```diff
@@ -43,11 +43,15 @@ def cmd_dependency_update(dry_run: str):
 
 #### Not Updated
 
-After merging this pull request, the following packages will **not** be on the latest version:
+{
+f"""After merging this pull request, the following packages will ** not ** be on the latest version:
```
Comment (felix11h) [inline actions_helper/main.py:47]:
```suggestion
f"""After merging this pull request, the following packages will **not** be on the latest version:
```

Is there any specific reason for adding the empty spaces here?

**Review by felix11h — APPROVED** [review]
Just the one comment, looks good to me otherwise! 

EDIT: Just making sure to ask if you tested this, now that I'm thinking about it, I'm not sure whether string within f-string is formatted correctly. Don't you need single quotes?

**Comment by felix11h** [comment]
Did you have a chance to look into this more, @IamAbbey?

File: actions_helper/main.py (hunk 1 above)
Comment (IamAbbey) [reply actions_helper/main.py:47]: No specific reason

**Comment by IamAbbey** [comment]
> Just the one comment, looks good to me otherwise!
> 
> EDIT: Just making sure to ask if you tested this, now that I'm thinking about it, I'm not sure whether string within f-string is formatted correctly. Don't you need single quotes?

works, it was tested

**Pull request merged**